src/com/palmergames/bukkit/towny/TownyUniverse.java

```diff
@@ -199,6 +204,11 @@ public void endWarEvent() {
         }
     }
     
+    public void finishTasks() {
+    	// Join into main thread for proper termination.
+		backupFuture.join();
```

Comment:
Might be useful to have a null-check before this, in-case something blows up before `performBackup` is called in `loadSettings()` and `backupFuture` would be null.